api/serializers/utils.py

```diff
@@ -166,3 +166,18 @@ def appro_to_percentages(representation, instance, remove_values=True):
 
 def float_or_none(value):
     return float(value) if value else None
+
+
+def match_sector_values(value):
```

Comment:
```suggestion
def clean_sector_values(value):
```
je ne vois pas de match dans cette fonction